lib/concepts/school_teacher/list.rb

```diff
@@ -15,11 +15,8 @@ def call(school:, token:)
 
       private
 
-      def list_teachers(school, token)
-        response = ProfileApiClient.list_school_teachers(token:, organisation_id: school.id)
-        user_ids = response.fetch(:ids)
-
-        User.from_userinfo(ids: user_ids)
+      def list_teachers(ids)
+        User.from_userinfo(ids:)
```

Comment:
💭 would it be possible for someone to call `list_teachers` and provide the ID of a student, and would that return them as it's only checking Users?

Reply:
If so, would the teacher/students having different properties cause things to fail?

Reply:
That is a good question, but actually this concept is a little half baked since we're not using this endpoint until the next ticket I've picked up (the bit around school members). Regardless it's not possible to query the api with a list of teachers, the `teacher_ids` param isn't actually defined, and it's only used internally by ClassMember::List.